--- a/czkawka_gui/ui/settings.ui	2022-11-30 18:47:44.710807848 -0500
+++ b/czkawka_gui/ui/settings.ui	2022-11-30 18:48:07.539874398 -0500
@@ -105,6 +105,7 @@
                             <property name="active">1</property>
                             <property name="focusable">1</property>
                             <property name="label" translatable="yes">Move deleted files to trash</property>
+                            <property name="sensitive">false</property>
                           </object>
                         </child>
                         <child>
--- a/czkawka_gui/src/connect_things/connect_button_delete.rs	2022-11-30 18:48:59.933206460 -0500
+++ b/czkawka_gui/src/connect_things/connect_button_delete.rs	2022-11-30 18:49:56.590937189 -0500
@@ -216,7 +216,7 @@
 }
 
 pub(crate) fn common_file_remove(sv: &SubView, check_button_settings_use_trash: &CheckButton, text_view_errors: &TextView, column_header: Option<i32>, file_remove: bool) {
-    let use_trash = check_button_settings_use_trash.is_active();
+    let use_trash = false && check_button_settings_use_trash.is_active();
 
     let model = sv.get_model();
 
--- a/czkawka_gui/src/saving_loading.rs	2022-11-30 19:05:16.727908825 -0500
+++ b/czkawka_gui/src/saving_loading.rs	2022-11-30 19:18:20.654609426 -0500
@@ -521,7 +521,8 @@
         settings.check_button_settings_hide_hard_links.set_active(default_config.hide_hard_links);
         settings.check_button_settings_use_cache.set_active(default_config.use_cache);
         settings.check_button_settings_save_also_json.set_active(default_config.use_json_cache_file);
-        settings.check_button_settings_use_trash.set_active(default_config.delete_to_trash);
+        //settings.check_button_settings_use_trash.set_active(default_config.delete_to_trash);
+        settings.check_button_settings_use_trash.set_active(false);
         settings.entry_settings_cache_file_minimal_size.set_text(&default_config.minimal_cache_size);
         settings
             .check_button_settings_similar_videos_delete_outdated_cache
@@ -650,7 +651,8 @@
         hide_hard_links: settings.check_button_settings_hide_hard_links.is_active(),
         use_cache: settings.check_button_settings_use_cache.is_active(),
         use_json_cache_file: settings.check_button_settings_save_also_json.is_active(),
-        delete_to_trash: settings.check_button_settings_use_trash.is_active(),
+        //delete_to_trash: settings.check_button_settings_use_trash.is_active(),
+        delete_to_trash: false,
         minimal_cache_size: settings.entry_settings_cache_file_minimal_size.text().to_string(),
         image_preview_image: settings.check_button_settings_show_preview_similar_images.is_active(),
         duplicate_preview_image: settings.check_button_settings_show_preview_duplicates.is_active(),
